content/docs/deployment/csmoperator/modules/authorization.md

```diff
@@ -102,23 +121,6 @@ To enable reporting of trace data with [Zipkin](https://zipkin.io/), use the `cs
   >__Note__:  
   > - This command will deploy the Authorization Proxy Server in the namespace specified in the input YAML file.
 
-5. Create the `karavi-auth-tls` secret using your own certificate or by using a self-signed certificate generated via cert-manager. 
-
-    If using your own certificate that is valid for each Ingress hostname, use this command to create the `karavi-auth-tls` secret:
-
-    ```bash
-
-    kubectl create secret tls karavi-auth-tls -n authorization --key <location-of-private-key-file> --cert <location-of-certificate-file>
```

Comment:
Is there another place where we are documenting how to create TLS from user-provided certificates?

Reply:
There are fields in the Authorization CR to specify user certificate files, so maybe it's self-explanatory and we don't need to document it.

Reply:
I don't believe it is documented elsewhere. 

That being said, this removal comes from a gitops branch for authorization (https://github.com/dell/csm-docs/commit/2d8a2b0166d254f0b9b470bb843404dd937201fc). @shaynafinocchiaro 

If needed, I can bring it back. Thoughts?

Reply:
We can leave it out as it should be clear in the CR where to specify certificate files.